Simplified/Reader2/AdobeDRM/AdobeDRMLicense.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AdobeDRMLicense.swift
+//  SimplyE
+//
+//  Created by Vladimir Fedorov on 13.05.2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+import R2Shared
+import R2Streamer
+
+/// DRM License for Adobe DRM
+class AdobeDRMLicense: DRMLicense {
+  
+  var publicationContainer: Container?
+  
+  /// DRM container for file
+  var adobeDRMContainer: AdobeDRMContainer?
+  
+  /// DRM license for file
+  /// - Parameter container: .epub file container
+  init(for container: Container) {
+    publicationContainer = container
+    let fileUrl = URL(fileURLWithPath: container.rootFile.rootPath)
+    let encryptionPath = "META-INF/encryption.xml"
+    do {
+      let data = try container.data(relativePath: encryptionPath)
+      adobeDRMContainer = AdobeDRMContainer(url: fileUrl, encryptionData: data)
+      if let errorMessage = adobeDRMContainer?.epubDecodingError {
+        NYPLErrorLogger.logError(withCode: .epubDecodingError, context: NYPLErrorLogger.Context.ereader.rawValue, message: errorMessage)
+      }
+    } catch {
+      NYPLErrorLogger.logError(withCode: .epubDecodingError, context: NYPLErrorLogger.Context.ereader.rawValue, message: error.localizedDescription)
```

Comment:
since you have it, you can also pass the caught `error` in the `metadata` field of the NYPLErrorLogger api, it will take care of logging it to Crashlytics for you.